### DISCUSSION TITLE
Non-adjusted close price. Is there a way to get?

### DISCUSSION THREAD

**Discussion by ZmicierGT** [ranaroussi/yfinance#1682] [Q&A] [answered]
Hi. Interesting, is there a way to get a non-adjusted close price values? As far as I understand, on Yahoo Finance the adjusted close is considered to be a close price adjusted to dividends and splits and a 'regular' close value is still adjusted to splits (but not to dividends).

Here is a short example of how I get the quotes:
```
import yfinance as yf
data = yf.Ticker('NKE')
data.history(start='2015-12-1', end='2015-12-30')
```
This way I get only one Close column and it is adjusted both for splits and dividends.

If I fetch the quotes this way:
`yf.download('NKE', start='2015-12-1', end='2015-12-30')`
I get both columns (Close and Adj Close). The first one is adjusted to splits only, the second one to splits and dividends.

Is there a way to get a truly unadjusted as traded close value?

Also interesting what happens if I get an intraday quotes?
`yf.download('NKE', interval='1m')`
What will happen if during the last week dividends come or split happens?

I appreciate any help?

**Comment by ValueRaider** [comment, marked as answer]
> Is there a way to get a truly unadjusted as traded close value?

Not split adjusted? Got to reverse adjustment yourself. Or use [yfinance_cache](https://github.com/ValueRaider/yfinance-cache#adjusting-price)
